Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -652,6 +652,14 @@ private void SetHandDataFromController(IMixedRealityController controller, IMixe
                         data.touchingRayOffset = handRayPt - SnapFingerToQuad(touchPosition);
                     }
                 }
+               
+                if (data.currentPointer is GGVPointer)
```

Comment:
Just for context, does `TryGetHandRayPoint` return false for the `WindowsMixedRealityGGVHand` controller on HoloLens 1?

Reply:
Also out of curiousity, should this be an else if? i.e. is the hand ray stuff even valid to check if this is a GGVPointer?

I guess maybe this is what @MenelvagorMilsom is maybe asking about

Reply:
It returns true in the editor or on HL1, but has same value for both hands. This start from zero brakes scale logic.

Reply:
This could be an option, but for readability it would put edge case as the first if check? is it better?
```suggestion
               if (data.currentPointer is GGVPointer)
                {
                    data.touchingInitialPt = SnapFingerToQuad(data.currentPointer.Position);
                }
               else if (TryGetHandRayPoint(controller, out Vector3 handRayPt) == true)
```

Reply:
I think since we're going to be checking this case whether it's first or second, it's neater to check it first if it allows us to use an else if.